src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -46,7 +46,7 @@ public static TransportExtensions<MsmqTransport> TransactionScopeOptions(this Tr
         /// </summary>
         public static FileRoutingTableSettings DistributeMessagesUsingFileBasedEndpointInstanceMapping(this TransportExtensions<MsmqTransport> config, string filePath)
         {
-            return config.UnicastRouting().Mapping.DistributeMessagesUsingFileBasedEndpointInstanceMapping(filePath);
+            return config.Routing().Mapping.DistributeMessagesUsingFileBasedEndpointInstanceMapping(filePath);
```

Comment:
I think this extension is the only place that would need to be modified if the change I proposed above happened. Seems reasonable to me to copy the relevant parts of the other method here, if we do in fact want to have this MSMQ-scoped version of this same call here.